tests/integration/features/servicemesh_feature_test.go

```diff
@@ -110,17 +69,20 @@ var _ = Describe("Service Mesh feature", func() {
 				applyErr := featuresHandler.Apply()
 
 				// then
-				Expect(applyErr).To(MatchError(ContainSubstring("customresourcedefinitions.apiextensions.k8s.io \"servicemeshcontrolplanes.maistra.io\" not found")))
+				Expect(applyErr).To(MatchError(ContainSubstring("failed to find the pre-requisite operator subscription \"servicemeshoperator\"")))
 			})
 		})
 
 		When("operator is installed", func() {
 			var smcpCrdObj *apiextensionsv1.CustomResourceDefinition
 
 			BeforeEach(func() {
-				// Create SMCP the CRD
+				// Create SM Operator Subscription
+				CreateSubscription(fixtures.OssmSubscription, "openshift-operators")
```

Comment:
The comment is redundant here, it's immediately visible from reading the code.

```suggestion
				CreateSubscription(fixtures.OssmSubscription, "openshift-operators")
```